addons/website/static/src/js/tours/homepage_edit_discard.js

```diff
@@ -0,0 +1,6 @@
+/** @odoo-module alias=website.homepage_edit_discard*/
+"use strict";
+
+import wTourUtils from "website.tour_utils";
+
+wTourUtils.homepageEditDiscard("homepage_edit_discard", []);
```

Comment:
Do we really want an entire file for this small tour ? Nothing wrong with that, I'm just wondering. @qsm-odoo What is your policy for tours ? Like one tour per file ?

Reply:
There is no fixed rule. In general, yes one tour per file I guess. But if they are very related or even part of the same python tour, of course makes sense to have multiples in the same file. But in the case of this PR @dhba-odoo:

- There are issues with the tour system at the moment (maybe this affects this) but `saveAs` is meant to override an existing tour, which I don't think is what you are doing here ?

- Why do we need an util for creating tours that click on the discard button at the end? Seems overkill.

- Don't you mean to create a python test out of this? And is not this supposed to be a test tour (not shown to users) (in which case, missing `test: true` parameter here + tour defined in the wrong folder ?)